internal/helper/teleport.go

```diff
@@ -0,0 +1,29 @@
+package helper
+
+import (
+	"context"
+
+	tc "github.com/gravitational/teleport/api/client"
+)
+
+var NewTeleportClient = func(ctx context.Context, identityFilePath string) (*tc.Client, error) {
```

Comment:
Can we please follow the same syntax used by the rest of the project and have this as a name function instead of a variable.

Also, it might make sense to have this as it's own module and use the standard `New` function to create a client.

Reply:
Thanks! Yeah, making it's own module makes sense.